rllib/BUILD

```diff
@@ -3200,6 +3209,15 @@ py_test(
     args = ["--as-test", "--framework=torch", "--stop-reward=80"]
 )
 
+py_test(
+    name = "examples/multi_agent_custom_policy_torch_w_rlm",
```

Comment:
"rlm" -> "rl_module"

Also: see my comment below: Can we use a command line arg here (`--use-rl-module`)?

Reply:
answered below.